webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -682,8 +682,10 @@ export const GlideTable: React.FC<GlideTableProps> = ({
               '',
             )}`;
             break;
+          case V1LocationType.TRAINING:
+            dataPath = `bestTrial.summaryMetrics.${currentColumn.column}`;
```

Comment:
fix: this path will return a MetricSummary object, rather than a value. I would suggest adding `.last` to the end of the path

sugg: we might want to handle validation metrics this way too

Reply:
We encode the possible paths of the MetricSummary in the columns api here, no? https://github.com/determined-ai/determined/pull/7320/files#diff-759959d3eba07f9028dad61e59aa27a4e2b7601f3424ead0036d17e7e2ce4cb5R244-R260

That said, this PR currently doesn't parse the training metrics into the decoded trial -- i'll do that now.